spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -36,14 +36,6 @@
 
 	private final Publisher publisher = new Publisher();
```

Comment:
These properties must have some JavaDoc description as well.

Reply:
Actually they must not. This will create a sub namespace and not a property (Or am I missing something?)

Reply:
Doh. Sure! We don't need JavaDocs on them, but I guess that won't hurt since it isn't clear what is the point of this property reading source code or JavaDocs of the class.